webpack/assets/javascripts/react_app/components/hosts/powerStatus/PowerStatusSelectors.js

```diff
@@ -0,0 +1,5 @@
+import { HOST_POWER_STATUS } from '../../../redux/consts';
+
+export const selectPowerStatusAPI = state => state.API[HOST_POWER_STATUS] || {};
```

Comment:
we have it in several other selectors actually,
to prevent doing something like `selectPowerStatusAPI(store) && selectPowerStatusAPI(store)[id]`, 

I heard about the optional chaining, looks promising :+1: